src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Testing/RunTestsHandler.cs

```diff
@@ -66,6 +66,42 @@ public async Task<RunTestsPartialResult[]> HandleRequestAsync(RunTestsParams req
         return progress.GetValues() ?? Array.Empty<RunTestsPartialResult>();
     }
 
+    /// <summary>
+    /// Format a timespan as a string similar to '5m 2s', omitting any value that is not present.
+    /// </summary>
+    internal static string GetShortTimespan(TimeSpan t)
+    {
+        var shortForm = "";
+        // Only output milliseconds if less than a second duration
+        if (t.Milliseconds > 0 && t.TotalSeconds < 1)
```

Comment:
`0ms` is acceptable for time span 0.

```suggestion
        if (t.TotalSeconds < 1)
```